web/src/engine/core-processor/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { CoreFactory, KM_CORE_STATUS } from './core-factory.js';
+import { type MainModule, type km_core_keyboard, type CoreKeyboardReturn } from './import/core/keymancore.js';
+export { MainModule, km_core_keyboard, CoreKeyboardReturn };
```

Comment:
```suggestion
export { MainModule as CoreMainModule, km_core_keyboard, CoreKeyboardReturn };
```

optional but may help for clarity elsewhere

Reply:
No longer relevant.